### ISSUE TITLE
Analytics KPI dashboard update request for My VA: Add total users and total pageviews to Domo dashboard

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#66654]
# Request an update to your KPI Dashboard  

## What this form is for

Use this ticket template to request an update to your existing KPI dashboard. To request a new product be added to our dashboards, please use [Request a New Standardized KPI Dashboard template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+kpi-dashboard&template=analytics-request-kpi-dashboard.md&title=Analytics+KPI+dashboard+request+for+%5BTeam%5D).

### Expected delivery timelines

Once you’ve submitted this ticket template, your KPI dashboard update request will be pulled into the Analytics team’s workflow the following Wednesday during Platform Sprint Planning or Refinement. We expect to deliver your completed dashboard within the next sprint. 

## Instructions for the VFS team
First, tell us what type of change you need to make:

- [ ] Change basic product info
- [ ] **Forms**: funnel changes
- [ ] **Content**: Add or remove pages included
- [X] Other

Could please add the following to the [My VA Domo dashboard](https://va-gov.domo.com/page/1167851935?userId=1322887837)? 

- The number of users who visit the My VA dashboard
- The total pageviews for My VA

Some relevant details:

- My VA is located at `www.va.gov/my-va`. However, there are some permutations of the URL as a result of redirects, eg. `/my-va/post-login=true`, so please filter for anything that starts with `/my-va`.
- For these two metrics, I don't need graphs that shows differences over time. A single number for each metric will do.

----

Next, use the original [KPI request template](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) or [(downloaded Excel version)](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/1586823275/KPI%20Dashboard%20Product%20Details%20Template%20-%20MAKE%20A%20COPY.xlsx?inst-v=ced1f4dd-e997-4aa2-9ae8-2cebe369b42f) you filled out to update your information. If you are not able to find that spreadsheet, reach out to the Analytics team on #vfs-platform-support for a new spreadsheet that reflects what we currently have on file.

- [ ] Make a copy of the [Google Sheet](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) [(or download Excel version)](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/1586823275/KPI%20Dashboard%20Product%20Details%20Template%20-%20MAKE%20A%20COPY.xlsx?inst-v=ced1f4dd-e997-4aa2-9ae8-2cebe369b42f)
- [ ] Complete worksheet as instructed
- [ ] Link your completed spreadsheet here: ______
- [ ] Check your sharing settings to make sure the Analytics team will be able to access it
- [ ] Add your team label to follow progress

**Additional Info**

- Please use the [KPI Framework](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/Analytics%20Playbook/va-gov-platform-analytics-kpi-framework.pdf) for more information on the metrics used in the KPI dashboards. 
- If you need any analytics support, read through the [How to request Analytics support](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guidelines.1586823275.html#Analyticscustomersupportguidelines-4.RequestaKPIDashboardinDomo).
- If you do not have Domo access, please [request access here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+access-request&template=analytics-request-google-analytics-domo-access.md&title=Request+access+to+Google+Analytics+and%2For+Domo).

**Comment by Samara-Strauss** [comment]
cc/ @ajakabcin I asked for the analytics team to add number of users who visit My VA to the Domo dashboard. We currently track sessions where users visit, the page, but not the number of users or total pageviews in Domo.

**Comment by michelle-dooley** [comment]
@Samara-Strauss I will review with team and get back to you with timing.

**Comment by michelle-dooley** [comment]
@Samara-Strauss - we are brining this into the sprint starting tomorrow.

**Comment by Samara-Strauss** [comment]
Thank you so much!

**Comment by michelle-dooley** [comment]
@Samara-Strauss the user and pageviews cards have been added to the dashboard. 

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMXpaQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3745164b3c9c300a59bbd44f2499f656cb07bfbc/image.png)

**Comment by Samara-Strauss** [comment]
Sorry to reopen this, but this isn't quite what I was looking for. I was hoping for a singular, cumulative total for pageviews for anything that begins with `va.gov/my-va/` and for users who visit anything that begins with `va.gov/my-va/`. To be clear, the URL needs to begin with `va.gov/my-va/`, not simply containt `my-va` anywhere in it. 

So URLs like this would contribute to these totals:

- www.va.gov/my-va/?postlogin=true
- www.va.gov/my-va/
- www.va.gov/?next=/my-va/

...and so on.

URLs like this WOULD NOT contribue to these totals:

- www.va.gov/disability/file-disability-claim-form-21-526ez/introduction?next=/my-va/
- www.va.gov/my-health/appointments?next=/my-va/
- www.va.gov/records/download-va-letters/letters/confirm-address?next=/my-va/

...and so on.


Let me know what questions you have.

**Comment by michelle-dooley** [comment]
@Samara-Strauss I updated the tables so they only show pages that begin with va.gov/my-va (and variations as described above). Take a look and let me know if this looks correct. I also added a total row at bottom but if you would rather just a total count I can remove the individual page paths, I just wanted you to see them and confirm the ones included were correct before I remove that detail. Let me know what you think. 

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMkR1QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--096f0bc480278a67f82c6c3d1d593bc5b16c8248/image.png)
